controllers/basicAuth_test.go

```diff
@@ -55,15 +55,15 @@ func Test_BasicLoginHandler_UserNotFound(t *testing.T) {
 		Return(nil, httperrors.AnError)
 	sessionService := mocksSessionService.NewSessionService(t)
 
-	controller := controllers.NewBasicAuthentificationController(
+	controller := controllers.NewBasicAuthenticationController(
 		logger,
 		userService,
 		sessionService,
 	)
 	response := httptest.NewRecorder()
 	request := httptest.NewRequest(
 		"POST",
-		"/v1/auth/basic/login",
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.